### PR TITLE
Adds Ration Packs to Sustenance Vendors

### PR DESCRIPTION
**Pull request by Kenionatus**
## About The Pull Request
Adds 12 ration packs to the list of available items in the sustenance vendor and at the same time reduces the number of available soggy tofu bars by the same amount. Whether that's necessary and whether soggy tofu even needs to remain in the vendor is up for discussion.

Adds fitting text to product_slogans (the ads that the machine occasionally speaks). Also moves some text from product_adds to product_slogans so that the machine speaks those. I don't think product_adds works any more anyway.

In the future, the sustenance vendor's description should probably be changed to reflect that it's not just for prisoners, considering it's in a public space on the Aetherwhisp. Maybe a subtype for public places would be good. Or we just change the description to something more neutral, not mentioning prisoners.

## Why It's Good For The Game
Ration packs are the thing that militaries feed to their soldiers if there isn't the capacity to make actual food. We're kind of a military server, so it makes sense to have them. The atlas even has them on the mess hall tables. The Aetherwhisp has a sustenance vendor in its dining area and this seems like the perfect place to me to distribute those rations.

## Changelog
:cl:
add: Added ration packs to sustenance vendors
/:cl:

### CODE REVIEW

File: code/modules/vending/sustenance.dm [hunk 1]
```diff
@@ -1,11 +1,12 @@
 /obj/machinery/vending/sustenance
 	name = "\improper Sustenance Vendor"
 	desc = "A vending machine which vends food, as required by section 47-C of the NT's Prisoner Ethical Treatment Agreement."
-	product_slogans = "Enjoy your meal.;Enough calories to support strenuous labor."
-	product_ads = "Sufficiently healthy.;Efficiently produced tofu!;Mmm! So good!;Have a meal.;You need food to live!;Have some more candy corn!;Try our new ice cups!"
+	product_slogans = "Enjoy your meal.;Enough calories to support strenuous labor.;Sufficiently healthy.;Efficiently produced tofu!;Mmm! So good!;Have a meal.;You need food to live!;Have some more candy corn!;Try our new ice cups!;Rememver: Ration packs save lifes, soldier!;Don't believe the lies about ration packs.; Ration packs: custom tailored to the Spaceman's taste!"
+	product_ads = "Sufficiently healthy.;Efficiently produced tofu!;Mmm! So good!;Have a meal.;You need food to live!;Have some more candy corn!;Try our new ice cups!;Rememver: Ration packs save lifes, soldier!;Don't believe the lies about ration packs.; Ration packs: custom tailored to the Spaceman's taste!"
```
Comment (Bokkiewokkie) [inline code/modules/vending/sustenance.dm:5]:
```suggestion
	product_slogans = "Enjoy your meal.;Enough calories to support strenuous labor.;Sufficiently healthy.;Efficiently produced tofu!;Mmm! So good!;Have a meal.;You need food to live!;Have some more candy corn!;Try our new ice cups!;Rememver: Ration packs save lifes, soldier!;Don't believe the lies about ration packs.; Ration packs: custom tailored to the Spaceman's taste!" //NSV13 added lines referencing ration packs
	product_ads = "Sufficiently healthy.;Efficiently produced tofu!;Mmm! So good!;Have a meal.;You need food to live!;Have some more candy corn!;Try our new ice cups!;Remember: Ration packs save lifes, soldier!;Don't believe the lies about ration packs.; Ration packs: custom tailored to the Spaceman's taste!" //NSV13 added lines referencing ration packs
```
Add comments to indicate we changed it from the base code.

File: code/modules/vending/sustenance.dm [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 	icon_state = "sustenance"
 	light_color = LIGHT_COLOR_BLUEGREEN
-	products = list(/obj/item/reagent_containers/food/snacks/tofu/prison = 24,
+	products = list(/obj/item/reagent_containers/food/snacks/tofu/prison = 12,
+					/obj/item/reagent_containers/food/snacks/rationpack = 12,
```
Comment (Bokkiewokkie) [inline code/modules/vending/sustenance.dm:9]:
```suggestion
	products = list(/obj/item/reagent_containers/food/snacks/tofu/prison = 12, //NSV13 halved available tofu
					/obj/item/reagent_containers/food/snacks/rationpack = 12, //NSV13 added ration packs
```
Ditto

**Review by Bokkiewokkie — CHANGES_REQUESTED** [review]

**Comment by Kenionatus** [comment]
Test fail looks very much unrelated to this PR.

**Review by covertcorvid — APPROVED** [review]
This makes the lizard sad, but it's a reasonable change

**Review by Bokkiewokkie — APPROVED** [review]

**Pull request merged**